frameworks/Rust/actix/benchmark_config.json

```diff
@@ -19,6 +19,44 @@
       "notes": "",
       "versus": ""
     },
+    "raw": {
+      "fortune_url": "/fortune",
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
```

Comment:
What is the difference between json and plaintext in the default profile, and this one?

Reply:
Other profiles uses high level api, this profile uses low-level api. It is similar to aspcore

Reply:
Ok, I was wondering if it was intentional to have other json and plaintext scenario, so it is, thanks. In aspnet we use 'raw' for the db access, it confused me ;)